.github/workflows/test.yml

```diff
@@ -19,66 +19,87 @@ jobs:
           - beta
           - nightly
           - 1.56.1
+    env:
+      RUSTFLAGS: "-C target-cpu=native -C opt-level=3"
```

Comment:
MSRV cargo does not know about profiles, so it was running with opt-level 0.

This is going to make out CI much faster :)